index.js

```diff
@@ -8,15 +8,16 @@ TODO
 var assign = require('object-assign');
 var debug = require('debug')('keystone-s3');
 var ensureCallback = require('keystone-storage-namefunctions/ensureCallback');
-var knox = require('knox');
+var AWS = require('aws-sdk');
 var nameFunctions = require('keystone-storage-namefunctions');
 var pathlib = require('path');
+var fs = require('fs');
 
 var DEFAULT_OPTIONS = {
-	key: process.env.S3_KEY,
-	secret: process.env.S3_SECRET,
-	bucket: process.env.S3_BUCKET,
-	region: process.env.S3_REGION || 'us-east-1',
+	accessKeyId: process.env.S3_KEY || process.env.AWS_ACCESS_KEY_ID,
+	secretAccessKey: process.env.S3_SECRET || process.env.AWS_SECRET_ACCESS_KEY,
+	region: process.env.S3_REGION || process.env.AWS_REGION,
+	bucket: process.env.S3_BUCKET || 'us-standard',
```

Comment:
Here allowing both aws default env var names and the previous env var names.. Should we go with just one?